### DISCUSSION TITLE
Increase `accessionNumber` field to 200 characters

### DISCUSSION THREAD

**Discussion by grantfitzsimmons** [specify/specify7#5712] [Feature Requests]
> We wanted to put in a request to increase the `accessionNumber` column in the Accession table from 60 to 200 characters so we can use our full accession names instead of our current auto-number solution.

**Requested By:** Dan Baker at CSIRO

**Comment by grantfitzsimmons** [comment]
> Here is the list of accessions that are longer than 60 characters, and my attempt at a clean-up - the interim would be to use a separate field to store the “full name” and use accession number as the “short name” so it’s at least human readable rather than a random number

![image](https://user-images.githubusercontent.com/37256050/212576180-e6e2c4fc-7dc6-480c-9ee8-10259a5a5973.png)

**Comment by g1rly-c0d3r** [comment]
Linked in https://github.com/specify/specify7/issues/8191
